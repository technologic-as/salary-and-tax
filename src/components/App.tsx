import React, { Fragment } from 'react';
import { GitHub } from './GitHub';
import { SalaryCalculations } from './SalaryCalculations';
import { SalaryParameters } from './SalaryParameters';
import { Summary } from './Summary';
import { SurplusGraph } from './SurplusGraph';
import { TaxCalculations } from './TaxCalculations';

export const AppComponent = () => (
  <Fragment>
    <SalaryCalculations />
    <TaxCalculations />
    <SurplusGraph />
    <Summary />
    <SalaryParameters />
    <GitHub />
  </Fragment>
);

export default AppComponent;
