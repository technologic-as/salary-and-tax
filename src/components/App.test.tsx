import mockComponent from 'identity-obj-proxy';
import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

jest.mock('./SalaryParameters', () => mockComponent);
jest.mock('./SalaryCalculations', () => mockComponent);
jest.mock('./TaxCalculations', () => mockComponent);
jest.mock('./Summary', () => mockComponent);
jest.mock('./SurplusGraph', () => mockComponent);
jest.mock('./GitHub', () => mockComponent);
jest.mock('./Ui', () => mockComponent);

describe('App', () => {
  it('should render', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
