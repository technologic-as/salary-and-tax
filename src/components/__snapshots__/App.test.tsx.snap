// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
Array [
  <SalaryCalculations />,
  <TaxCalculations />,
  <SurplusGraph />,
  <Summary />,
  <SalaryParameters />,
  <GitHub />,
]
`;
